client-mobile/src/containers/ShoppingListsContainer/index.tsx

```diff
@@ -1,14 +1,34 @@
+import { useCreateShoppingList } from "@client/api/mutations/useCreateShoppingList"
 import { IconName } from "@client/components/atoms/Icon/@types"
 import { Button } from "@client/components/molecules/Button"
 import { TilesList } from "@client/components/molecules/TilesList"
+import { Path } from "@client/routing/paths"
+import { router } from "expo-router"
 
-export const ShoppingListsContainer: React.FC = () => (
-  <>
-    <TilesList
-      emptyResultDescription="Shopping lists will appear here"
-      emptyResultIcon={IconName.List}
-      title="Shopping lists"
-    />
-    <Button iconName={IconName.Plus} text="Add new shopping list" />
-  </>
-)
+export const ShoppingListsContainer: React.FC = () => {
+  const { isLoading, mutateAsync: createShoppingList } = useCreateShoppingList({
```

Comment:
We should use `mutate`, not `mutateAsync` here, because `mutateAsync` is a Promise, which might throw unexpected errors.
```suggestion
  const { isLoading, mutate: createShoppingList } = useCreateShoppingList({
```